### ISSUE TITLE
uninitialized constant Process::CLOCK_PROCESS_CPUTIME_ID with JRuby 

### ISSUE THREAD

**Issue by yahonda** [rsim/oracle-enhanced#1735]
### Steps to reproduce
```ruby
$ bundle exec rspec ./spec/active_record/connection_adapters/oracle_enhanced/connection_spec.rb:37
```

### Expected behavior
It should pass.

### Actual behavior
```ruby
$ bundle exec rspec ./spec/active_record/connection_adapters/oracle_enhanced/connection_spec.rb:37
Coverage may be inaccurate; set the "--debug" command line option, or do JRUBY_OPTS="--debug" or set the "debug.fullTrace=true" option in your .jrubyrc
/home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/simplecov-03a9f7fde44a/lib/simplecov.rb:53: warning: tracing (e.g. set_trace_func) will not capture all events without --debug flag
==> Loading config from ENV or use default
==> Running specs with JRuby version 9.2.0.0
/home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/ruby-plsql-e0b3e01204a8/lib/plsql/jdbc_connection.rb:225: warning: constant ::Fixnum is deprecated
/home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/ruby-plsql-e0b3e01204a8/lib/plsql/jdbc_connection.rb:226: warning: constant ::Bignum is deprecated
==> Effective ActiveRecord version 6.0.0.alpha
Run options: include {:locations=>{"./spec/active_record/connection_adapters/oracle_enhanced/connection_spec.rb"=>[37]}}
F

Failures:

  1) OracleEnhancedAdapter establish connection should use database default cursor_sharing parameter value exact by default
     Failure/Error: super

     NameError:
       uninitialized constant Process::CLOCK_PROCESS_CPUTIME_ID
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/instrumenter.rb:134:in `now_cpu'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/instrumenter.rb:75:in `start!'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/subscriber.rb:83:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/log_subscriber.rb:80:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/fanout.rb:117:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/fanout.rb:44:in `block in start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/fanout.rb:44:in `each'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/fanout.rb:44:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/instrumenter.rb:35:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activesupport/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb:574:in `log'
     # ./lib/active_record/connection_adapters/oracle_enhanced/dbms_output.rb:36:in `log'
     # ./lib/active_record/connection_adapters/oracle_enhanced/database_statements.rb:24:in `exec_query'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:465:in `select'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:57:in `select_all'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract/query_cache.rb:101:in `select_all'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:81:in `select_rows'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/bundler/gems/rails-e03add224899/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:69:in `select_value'
     # ./spec/active_record/connection_adapters/oracle_enhanced/connection_spec.rb:36:in `block in (root)'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:254:in `instance_exec'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:254:in `block in run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:500:in `block in with_around_and_singleton_context_hooks'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:457:in `block in with_around_example_hooks'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/hooks.rb:466:in `block in run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/hooks.rb:604:in `run_around_example_hooks_for'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/hooks.rb:466:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:457:in `with_around_example_hooks'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:500:in `with_around_and_singleton_context_hooks'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example.rb:251:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example_group.rb:628:in `block in run_examples'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example_group.rb:624:in `map'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example_group.rb:624:in `run_examples'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/example_group.rb:590:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:118:in `block in run_specs'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:118:in `map'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:118:in `block in run_specs'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/configuration.rb:1926:in `with_suite_hooks'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:113:in `block in run_specs'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:79:in `report'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:112:in `run_specs'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:87:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:71:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:45:in `invoke'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/rspec-core-3.7.1/exe/rspec:4:in `<main>'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/bin/rspec:1:in `load'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/bin/rspec:1:in `<main>'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/bin/rspec:23:in `load'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/bin/rspec:23:in `(root)'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli/exec.rb:1:in `(root)'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli/exec.rb:74:in `kernel_load'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli/exec.rb:28:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli.rb:424:in `exec'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli.rb:27:in `dispatch'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `block in start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/cli.rb:18:in `start'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/exe/bundle:30:in `<main>'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/friendly_errors.rb:124:in `load'
     # /home/yahonda/.rbenv/versions/jruby-9.2.0.0/lib/ruby/gems/shared/gems/bundler-1.16.3/lib/bundler/friendly_errors.rb:124:in `<main>'

Finished in 0.72651 seconds (files took 2.32 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/active_record/connection_adapters/oracle_enhanced/connection_spec.rb:37 # OracleEnhancedAdapter establish connection should use database default cursor_sharing parameter value exact by default

Coverage report generated for RSpec to /home/yahonda/git/oracle-enhanced/coverage. 0 / 2140 LOC (0.0%) covered.
$
```
### System configuration
**Rails version**: master branch including https://github.com/rails/rails/commit/f0c917c7d491/

**Oracle enhanced adapter version**: master branch

**Ruby version**:jruby 9.2.0.0 (2.5.0) 2018-05-24 81156a8 Java HotSpot(TM) 64-Bit Server VM 25.171-b11 on 1.8.0_171-b11 [linux-x86_64]

**Oracle Database version**:Oracle Database 12c Enterprise Edition Release 12.2.0.1.0 - 64bit Production

**Comment by yahonda** [comment]
It looks it is due to https://github.com/rails/rails/pull/33449

**Comment by yahonda** [comment]
It has been addressed by rails/rails#33468
